hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapreduce/TestSyncTable.java

```diff
@@ -463,26 +490,28 @@ private void writeTestData(TableName sourceTableName, TableName targetTableName)
     int numRows = 100;
     int sourceRegions = 10;
     int targetRegions = 6;
+    if (ArrayUtils.isEmpty(timestamps)) {
```

Comment:
If you make `timestamps` a `Pair`, that would make this a little more terse (avoid the ability for the caller to give you a single timestamp, or three timestamps)

Reply:
Decided to use varargs so that I didn't need to change other existing methods already calling writeTestData.

Reply:
Ok, sounds good.